README.md

```diff
@@ -743,6 +761,8 @@ export default {
 }
 ```
 
+To support backwards-compatibility, monorepo features require multiple _lint-staged_ configuration files present in the git repo. If your monorepo setup requires running _lint-staged_ in only one of the packages, you can either add an "empty" _lint-staged_ configuration to the root of the repo (so that there's two configs in total), or alternatively run _lint-staged_ with the `--cwd` option pointing to your package directory (for example, `lint-staged --cwd packages/frontend`).
```

Comment:
Wow I didn’t know that! I’m wondering why it’s a requirement after all and what happens if I have one root config?

Reply:
What do you mean? If there's only one config in the root, then that applies to the entire git repo.

Maybe this explanation is not clear enough. By default all commands run in the git repo root, but with multiple configs they will run separately in the directories of each config file.

...now I see, this might be a bit confusing: 

> If your monorepo setup requires running _lint-staged_ in only one of the packages...

what might be better would be:

> If you absolute want to use monorepo features and run _lint-staged_ in only one of the packagess...

Reply:
How about I remove these README commits from this PR and create another one?

Reply:
I dropped the two `docs` commits, let's figure out some better wording to make them clearer. 👍

Reply:
I think the last proposal you made was much better and clear enough. Let’s add it.

Reply:
I’m fine with a separate PR too

Reply:
Let's do a separate. You'll have to re-approve the MR.

Reply:
Followup: https://github.com/okonet/lint-staged/pull/1234